test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -44,12 +44,25 @@ build_branch()
         git clone --quiet https://github.com/wiredtiger/wiredtiger.git "$1"
         cd "$1"
         git checkout --quiet "$1"
-
-        config=""
-        config+="--enable-snappy "
-        config+="--disable-standalone-build "
-        (sh build_posix/reconf &&
-            ./configure $config && make -j $(grep -c ^processor /proc/cpuinfo)) > /dev/null
+        if [ "${build_sys[$1]}" == "cmake" ]; then
+            . ./test/evergreen/find_cmake.sh
+            config=""
+            config+="-DENABLE_SNAPPY=1 "
+            config+="-DWT_STANDALONE_BUILD=0 "
+            (mkdir build && cd build &&
+                $CMAKE $config ../. && make -j $(grep -c ^processor /proc/cpuinfo)) > /dev/null
```

Comment:
Can we use `ninja` here?

Reply:
Same reason discussed in https://github.com/wiredtiger/wiredtiger/pull/7317#discussion_r769090697
Rather stick with `make` here.